pkg/internal/goexec/structmembers.go

```diff
@@ -201,16 +202,20 @@ func structMemberOffsetsFromDwarf(data *dwarf.Data) (FieldOffsets, map[string]st
 			continue
 		}
 		attrs := getAttrs(entry)
-		typeName := attrs[dwarf.AttrName].(string)
-		if structMember, ok := structMembers[typeName]; !ok {
+		typeName, ok := attrs[dwarf.AttrName]
```

Comment:
Had to verify that the name attribute exists because with the `-linkmode external -extldflags "-static"` we find some DWARF attributes whose Name attribute is `nil`.